firefox-ios/firefox-ios-tests/Tests/XCUITests/InactiveTabsTest.swift

```diff
@@ -67,6 +67,9 @@ final class InactiveTabsTest: BaseTestCase {
 
         // Return to tabs tray
         navigator.goto(TabTray)
+        app.otherElements["Tabs Tray"].swipeDown()
+        app.otherElements["Tabs Tray"].swipeDown()
+        app.otherElements["Tabs Tray"].swipeDown()
```

Comment:
The active tab is appended to the bottom. On iPhone, we need to scroll all the way up to see the inactive tabs.

Reply:
should we add a condition for iPhone then or will this work also on iPad?

Reply:
The swiping is not necessary on iPad since all tabs are displayed, but the swiping does not hurt.